pyvista/plotting/plotter.py

```diff
@@ -2095,12 +2102,12 @@ def pickable_actors(self):  # numpydoc ignore=RT01
         ]
 
     @pickable_actors.setter
-    def pickable_actors(self, actors=None):  # numpydoc ignore=GL08
+    def pickable_actors(self, actors=None) -> None:  # numpydoc ignore=GL08
         actors = [] if actors is None else actors
-        if isinstance(actors, _vtk.vtkActor):
+        if isinstance(actors, Actor):
```

Comment:
Hmmmm, maybe the suggestions I made were not ideal.  I think I see that we allow users to pass in a `_vtk.vtkActor` directly, so the typing was correct before.  Sorry about that.  We shouldn't make this change.